### PR TITLE
Supporting android

### LINKED ISSUE #28: Unsupported for android platform
Android is based on Linux.
In theory, it should be used.

### PR DESCRIPTION
**Pull request by lcron**
fixes #28

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ var os = require('os')
 var path = require('path')
 
 var platform = os.platform()
+platform = platform == 'android' ? 'linux' : platform
```
Comment (derhuerst) [inline index.js]: [`os.platform()` doesn't return `'android'`.](https://nodejs.org/docs/latest-v8.x/api/os.html#os_os_platform)

**Comment by lcron** [comment]
> Did you check if this actually works? I think it can't.

At first, I was trying to eliminate the warning, but then I found out that I was wrong. My project went into endless waiting when it was built, and the binary files could not be executed.

**Comment by lcron** [comment]
This is the text from the document.

> The value 'android' may also be returned if the Node.js is built on the Android operating system. However, Android support in Node.js is considered to be experimental at this time.

This is the result of my mobile phone test.

**Comment by derhuerst** [comment]
>> This is the text from the document.
>
> The value 'android' may also be returned if the Node.js is built on the Android operating system. However, Android support in Node.js is considered to be experimental at this time.

I missed that.

Can you confirm that `ffmpeg-static` for Linux runs on Android? Have you tried both `arm` and `arm64`?

**Comment by derhuerst** [comment]
@lcron Please confirm that `ffmpeg` for Linux runs on Android. Please test it e.g. by converting a small file.

**Comment by derhuerst** [comment]
Closing because of inactivity. Please reopen if you have confirmed that `ffmpeg` works on Android.

**Pull request closed without merging**